nck/utils/date_handler.py

```diff
@@ -63,3 +63,24 @@ def get_date_start_and_date_stop_from_date_range(date_range: str) -> Tuple[date,
     """
     current_date = date.today()
     return DEFAULT_DATE_RANGE_FUNCTIONS[date_range](current_date)
+
+
+def build_date_range(start_date, end_date, date_range):
```

Comment:
I'm the first one to add these when it is not really necessary so you can keep it the way it is. Nevertheless, I think the meaning of the variable is obvious while types may not be (date or datetime, that is the question). 
```suggestion
def build_date_range(start_date: date, end_date: date, date_range: str) -> Tuple[datetime, datetime]:
```
By doing so you could therefore remove the content of your docstring which may not really bring value here once the types have been specified using type hinting like I suggested. Again, your work is fine, this is just a simple suggestion you can keep it like it is now!

Reply:
Here the goal is really to delete the docstring which doesn't bring value to me except the types of arguments but you can specify them in an even more explicit way thanks to type hinting.